### PR TITLE
Add Share 7.0.1.2 image

### PR DESCRIPTION
**Pull request by WimCrols**

### CODE REVIEW

File: 2share/7.0/community-7.0.1/overload.gradle [hunk 1]
```diff
@@ -3,13 +3,13 @@ ext {
             version: [
                     major: 7,
                     minor: 0,
-                    rev: '0'
+                    rev: '1'
```
Comment (vierbergenlars) [inline 2share/7.0/community-7.0.1/overload.gradle]: Needs to have a label added.

See https://github.com/xenit-eu/docker-share/commit/768e4771142cadafd35859e88a902f91e37b4132 for example

**Review by vierbergenlars — CHANGES_REQUESTED** [review]

File: 1skeleton/7.0/overload.gradle [hunk 2]
```diff
@@ -3,8 +3,9 @@ ext {
 			version: [
 					major: 7,
 					minor: 0,
-					rev: 0
+					rev: 1,
+					label: 2
```
Comment (vierbergenlars) [inline 1skeleton/7.0/overload.gradle:7]: This skeleton does not need to be changed. A skeleton only contains the supporting files (which remain the same for a release series), so there is no need to add a specific version for them.

**Review by vierbergenlars — CHANGES_REQUESTED** [review]

File: Changelog.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [ALFREDOPS-687] Add image for 6.2.2.
 * [DOCKER-275] Introduce variables for direct communication share-alfresco. Make sure that ALFRESCO_INTERNAL_HOST is set to the correct name of the alfresco service/container.
 * [DEVEM-478] Share 7.0.0
+* [BNPPFALF7-6] Share 7.0.1.2
```
Comment (vierbergenlars) [inline Changelog.md:15]: Please don't reference customer jira tickets in public repositories

File: 1skeleton/7.0/overload.gradle (hunk 2 above)
Comment (WimCrols) [reply 1skeleton/7.0/overload.gradle:7]: Thanks for the explanation. At what point _does_ it need a change then? Major versions?

File: Changelog.md (hunk 3 above)
Comment (WimCrols) [reply Changelog.md:15]: To rename branch too following convention, I will start a new PR #21 then and close this one.

**Pull request closed without merging**